lms/djangoapps/courseware/views/index.py

```diff
@@ -341,7 +342,7 @@ def _create_courseware_context(self):
             'xqa_server': settings.FEATURES.get('XQA_SERVER', "http://your_xqa_server.com"),
             'bookmarks_api_url': reverse('bookmarks'),
             'language_preference': self._get_language_preference(),
-            'disable_optimizely': True,
+            'disable_optimizely': not WaffleSwitchNamespace('RET').is_enabled('enable_optimizely_in_courseware'),
```

Comment:
What's the point of namespace? It's obviously an edX customization, but do we actually need to use it here?